src/dnddiceroller/DiceTower.java

```diff
@@ -25,4 +25,23 @@ public DiceTower() {
   public DiceTower(List dice) {
 	this.dice = dice;
   }
+
+  public void dropDice() {
+        for (int i = 0; i < PANEL_COUNT; i++) {
+            for (int j = 0; j < dice.size(); j++) {
+                Die temp = dice.get(j);
```

Comment:
love it. `temp` isn't a great name but I like you pulling it into a variable. You could also have done
```
for(Dice item : this.dice){
  item.roll();
}
```